metricflow/sql_clients/databricks.py

```diff
@@ -87,17 +101,27 @@ def from_connection_details(url: str, password: Optional[str]) -> DatabricksSqlC
         except ValueError as e:
             # If any errors in parsing URL, show user what expected URL looks like.
             raise ValueError(
-                f"Unexpected format for MF_SQL_ENGINE_URL. Expected: `{dialect}://<HOST>:443;HttpPath=<HTTP PATH>`."
+                f"Unexpected format for MF_SQL_ENGINE_URL. Expected: `{dialect}://<HOST>:443;HttpPath=<HTTP PATH>` "
+                f"or optionally `{dialect}://<HOST>:443;HttpPath=<HTTP PATH>;HttpPathRename=<HTTP PATH FOR RENAMES>`."
             ) from e
 
         if not password:
             raise ValueError(f"Password not supplied for {url}")
 
-        return DatabricksSqlClient(host=parsed_url.host, http_path=http_path, access_token=password)
+        return DatabricksSqlClient(
+            host=parsed_url.host,
+            http_path=http_path,
+            access_token=password,
+            http_path_for_table_renames=http_path_for_table_renames,
+        )
 
-    def get_connection(self) -> sql.client.Connection:
+    def get_connection(self, table_rename: bool = False) -> sql.client.Connection:
         """Get connection to Databricks cluster/warehouse."""
-        return sql.connect(server_hostname=self.host, http_path=self.http_path, access_token=self.access_token)
+        return sql.connect(
+            server_hostname=self.host,
+            http_path=self.http_path_for_table_renames if table_rename else self.http_path,
```

Comment:
Do we want to do this here? The pro is anybody using this as a standalone client can actually do so even if Databricks throws that error in their face, but that's not really a thing we support.